source/agora/node/FullNode.d

```diff
@@ -149,6 +157,18 @@ public class FullNode : API
             this.storage, this.enroll_man, this.pool, onValidatorsChanged);
         this.exception = new RestException(
             400, Json("The query was incorrect"), string.init, int.init);
+
+        // Make `BlockExternalizedHandler` from config
+        foreach (block_externalize_address;
+            config.event_handlers.block_externalized_handler_addresses)
+            this.block_handlers ~= this.network.getBlockExternalizedHandler
+                (block_externalize_address);
```

Comment:
```suggestion
            this.block_handlers ~= this.network
                .getBlockExternalizedHandler(block_externalize_address);
```

There is a reason I suggest this. It's to make it easier to find function calls across the codebase.

Reply:
Yes, it looks much better.